atlassian_jwt_auth/contrib/tests/utils.py

```diff
@@ -1,20 +1,21 @@
 import atlassian_jwt_auth
 
 
-class StaticPublicKeyRetriever(object):
```

Comment:
The main problem with the `@requires_asap` is that you can't modify the decorator at runtime, only a setting  allows for that.